vendor_skins/Evercast/CI/before-script-osx.sh

```diff
@@ -3,14 +3,16 @@ export PATH=/usr/local/opt/ccache/libexec:$PATH
 mkdir build
 cd build
 cmake \
--DDepsPath=/tmp/obsdeps \
+-DENABLE_SCRIPTING=OFF \
+-DDepsPath=~/obsdeps \
 -DCMAKE_BUILD_TYPE=RELEASE \
 -DCMAKE_INSTALL_PREFIX=/opt/ebs \
--DVLCPath=$PWD/../../vlc-master \
--DQTDIR=/usr/local/Cellar/qt/5.10.1 \
--DCMAKE_OSX_DEPLOYMENT_TARGET=10.10 \
+-DVLCPath=~/vlc-3.0.4 \
```

Comment:
Interesting, this bundles VLC huh?  TIL

Reply:
Funny you mention it - the bundled lib still needs VLC to be installed or it won't load, which makes sense, since it's there to use VLC as a video source.